src/debugTools/renderer.cpp

```diff
@@ -10,6 +10,67 @@ Renderer::~Renderer() {
     cv::destroyAllWindows();
 }
 
```

Comment:
To get the rectangle to render around the detecetd cards we just need to pass the DetectedCards vector to the render function and loop through them rendering the cardRectangle property, similar to the render function you added at the top. 

At the moment this doesn't compile because or 10+ errors